src/i18n/en/translations.json

```diff
@@ -9,8 +9,8 @@
   "Active repository:":                  "Active repository:",
   "Adapter configuration":               "Adapter configuration",
   "Adapters":                            "Adapters",
-  "Adapters from this Group installed":  "Adapters from this Group installed",
```

Comment:
Eventuell müsste es "Installed adapters from this group" heißen. Kenne den Kontext nicht